<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI Semibold</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="subtitle2">
        <property name="text">
         <string>飲料</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="black_tea">
        <property name="text">
         <string>紅茶 $15</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="green_tea">
        <property name="text">
         <string>綠茶 $15</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cola">
        <property name="text">
         <string>可樂 $20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bubble_tea">
        <property name="text">
         <string>珍珠奶茶 $50</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="subtitle1">
        <property name="text">
         <string>主餐</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="omurice">
        <property name="text">
         <string>蛋包飯 $90</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="pasta">
        <property name="text">
         <string>義大利麵 $110</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="risotto">
        <property name="text">
         <string>義式燉飯 $110</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="beef_noodle">
        <property name="text">
         <string>牛肉麵 $130</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="subtitle3">
        <property name="text">
         <string>付費方式</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cash">
        <property name="text">
         <string>現金 $10 off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="card">
        <property name="text">
         <string>刷卡 10% off</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>點餐系統</string>
      </property>
      <property name="indent">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
